libraries/AP_HAL_ChibiOS/hwdef/KakuteF4/hwdef.dat

```diff
@@ -70,12 +70,17 @@ define HAL_BATT_CURR_SCALE 17.0
 # order of UARTs (and USB)
 SERIAL_ORDER OTG1 USART6 USART1 UART4 UART5 USART3
 
-# rcinput is PB11
+# RC input defaults to timer capture
 PB11 TIM2_CH4 TIM2 RCININT PULLDOWN LOW
-PB10 USART3_TX USART3
+
+# alternative RC input using USART3 serial port,
+PB10 USART3_TX USART3 NODMA
+PB11 USART3_RX USART3 NODMA ALT(1)
```

Comment:
You will need to fix the KakuteF4Mini defn since it defines this as well.

Reply:
You really need DMA for RC input

Reply:
It took me a while to find the hw defs of betaflight after they reworked their structure.
It may be a good idea to leave the original board untouched and create a new board config, if that is allowed.
https://github.com/betaflight/unified-targets/blob/master/configs/default/HBRO-KAKUTEF4V2.config

From what I have seen, there is UART4 new, the UART5RX for ESC Telemetry and some PINS changed as well.